utils/build/docker/ruby/rails40.Dockerfile

```diff
@@ -12,6 +12,6 @@ RUN /binaries/install_ddtrace.sh
 
 ENV DD_TRACE_HEADER_TAGS=user-agent
 
-RUN echo "#!/bin/bash\nbundle exec thin start -p 7777" > app.sh
+RUN echo "#!/bin/bash\nbundle exec rake db:create db:migrate db:seed\nbundle exec thin start -p 7777" > app.sh
```

Comment:
Is there a reason for Having `rake db:create db:migrate db:seed` at runtime rather during the build ?